### ISSUE TITLE
Unable to fetch a list of sections error

### ISSUE THREAD

**Issue by attackant** [alleyinteractive/apple-news#1056]
### Description of the bug

We are getting an error :

```
Unable to fetch a list of sections.
If you need assistance, please reach out for support on WordPress.org or GitHub.

```

### Steps To Reproduce

1. On a clean install of WP visit `/wp-admin/admin.php?page=apple_news_index`
2. We also get this when changing options in the Apple News component in the Gutenberg sidebar and hit save.

### Additional Information

This seems to be intermittent.

**Comment by attackant** [comment]
Fixed in #1049

**Comment by alkeshfinixio** [comment]
We have the Publish to Apple News plugin Version 2.4.6 installed on our wordpress version 6.7 and getting the following error:

**Unable to fetch a list of sections.
If you need assistance, please reach out for support on [WordPress.org](https://wordpress.org/support/plugin/publish-to-apple-news) or [GitHub](https://github.com/alleyinteractive/apple-news/issues).**


While opening the url /wp-admin/admin.php?page=apple_news_index we are getting the Unable to fetch a list of sections  error, even if we open our edit page option then also we get this error
